### PR TITLE
Fixes #7479: OpenSSH server configuration does not reload configurations

### PR DESCRIPTION
**Pull request by ncharles**

### CODE REVIEW

**Comment by ncharles** [comment]
http://www.rudder-project.org/redmine/issues/7479

**Comment by jooooooon** [comment]
Looks good to me!

**Comment by jooooooon** [comment]
In the long run it would be preferable to call ncf's service_restart bundle, though.

**Comment by peckpeck** [comment]
OK, merging

File: techniques/systemSettings/remoteAccess/sshConfiguration/5.0/main.st [hunk 1]
```diff
@@ -50,7 +50,7 @@ bundle agent rudder_openssh_server_configuration(class_prefix, service_name, par
 
     aix::
       "rudder_openssh_server_startup_command"
-        string => "/usr/bin/startsrc -s sshd";
+        string => "/usr/bin/stopsrc -s sshd && until /usr/bin/lssrc -s sshd | ${paths.grep} -q inoperative; do ${paths.perl} -e 'select(undef,undef,undef,.25)'; done; /usr/bin/startsrc -s sshd";
```
Comment (ncharles) [inline techniques/systemSettings/remoteAccess/sshConfiguration/5.0/main.st]: ha, && is invalid

**Pull request merged**